node/runtime/src/lib.rs

```diff
@@ -240,44 +236,49 @@ impl session::historical::Trait for Runtime {
 	type FullIdentificationOf = staking::ExposureOf<Runtime>;
 }
 
-parameter_types! {
-	pub const SessionsPerEra: sr_staking_primitives::SessionIndex = 3;
-	// Mainnet genesis bonding duration - number of eras to bond where eras are 1 hour long
-	pub const BondingDuration: staking::EraIndex = 2;
+pallet_staking_reward_curve::build! {
+	const REWARD_CURVE: PiecewiseLinear<'static> = curve!(
+		min_inflation: 0_025_000,
+		max_inflation: 0_100_000,
+		ideal_stake: 0_500_000,
+		falloff: 0_050_000,
+		max_piece_count: 40,
+		test_precision: 0_005_000,
+	);
 }
-srml_staking_reward_curve::build! {
-        const I_NPOS: PiecewiseLinear<'static> = curve!(
-                min_inflation: 0_025_000,
-                max_inflation: 0_100_000,
-                ideal_stake: 0_800_000,
-                falloff: 0_050_000,
-                max_piece_count: 40,
-                test_precision: 0_005_000,
-        );
+
+parameter_types! {
+	pub const SessionsPerEra: sr_staking_primitives::SessionIndex = 6;
+	pub const BondingDuration: staking::EraIndex = 24 * 28;
+	pub const SlashDeferDuration: staking::EraIndex = 24 * 7; // 1/4 the bonding duration.
+	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
 }
 
 impl staking::Trait for Runtime {
 	type Currency = Balances;
 	type Time = Timestamp;
 	type CurrencyToVote = CurrencyToVoteHandler;
-	type OnRewardMinted = ();
+	type RewardRemainder = Treasury;
 	type Event = Event;
 	type Slash = Treasury; // send the slashed funds to the treasury.
 	type Reward = (); // rewards are minted from the void
 	type SessionsPerEra = SessionsPerEra;
 	type BondingDuration = BondingDuration;
+	type SlashDeferDuration = SlashDeferDuration;
+	/// A super-majority of the council can cancel the slash.
+	type SlashCancelOrigin = collective::EnsureProportionAtLeast<_3, _4, AccountId, CouncilCollective>;
 	type SessionInterface = Self;
-        type RewardCurve = RewardCurve;
+	type RewardCurve = RewardCurve;
 }
 
 parameter_types! {
-	pub const LaunchPeriod: BlockNumber = 3 * 24 * 60 * MINUTES;
-	pub const VotingPeriod: BlockNumber = 3 * 24 * 60 * MINUTES;
-	pub const EmergencyVotingPeriod: BlockNumber = 1 * 24 * 60 * MINUTES;
+	pub const LaunchPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
```

Comment:
I think we should use a 3-day launch period, 3-day voting period, 24-hour enactment period, 24-hour emergency voting period, and 3-day cooloff period for testnet.

Reply:
will change.